pygfx/renderers/wgpu/shaders/backgroundshader.py

```diff
@@ -1,10 +1,18 @@
 import wgpu  # only for flags/enums
 
-from . import register_wgpu_render_function, WorldObjectShader, Binding, RenderMask
-from ._utils import to_texture_format, GfxSampler, GfxTextureView
-from ...objects import Background
-from ...materials import BackgroundMaterial, BackgroundImageMaterial
-from ...resources import Texture
+from ....objects import Background
+from ....materials import BackgroundMaterial, BackgroundImageMaterial
+from ....resources import Texture
```

Comment:
These four dots hurt my eyes a bit. I thought about collapsing `renderers/wgpu/`  into `wgpurenderer/` to reduce one level of nesting. I half-like that idea, but when I think about possibly having more renderers in the future, I very much like the current structure ...